modals/authentication.js

```diff
@@ -0,0 +1,7 @@
+const mongoose = require('mongoose');
```

Comment:
The folder name has a typo -- it should be "models."

Reply:
Fixed.